src/components/20-molecules/accordion/index.js

```diff
@@ -5,11 +5,51 @@ import expandMore from '@axa-ch/materials/icons/material-design/expand_more.svg'
 
 import { classMap } from 'lit/directives/class-map.js';
 import { defineVersioned } from '../../../utils/component-versioning';
-import applyDefaults from '../../../utils/apply-defaults';
 import { sanitizeSVG } from '../../../utils/sanitize';
+import applyDefaults from '../../../utils/apply-defaults';
 import fireCustomEvent from '../../../utils/custom-event';
+import createRefId from '../../../utils/create-ref-id';
 import styles from './index.scss';
 
+const collapseContent = el => {
+  // get the height of the element's inner content, regardless of its actual size
+  const contentHeight = el.scrollHeight;
+  // temporarily disable all css transitions
+  const elementTransition = el.style.transition;
```

Comment:
You are **saving** the existing transition here, to be used in a closure under rAF below. So, name it accordingly, e.g. savedElementTransition or savedTransition.